modules/kafka/kafka-dest-driver.c

```diff
@@ -333,34 +442,39 @@ kafka_dd_init(LogPipe *s)
   if (!log_threaded_dest_driver_init_method(s))
     return FALSE;
 
-  if (!self->topic_name)
+  if (!self->topic_name->template)
     {
       msg_error("kafka: the topic() argument is required for kafka destinations",
                 evt_tag_str("driver", self->super.super.super.id),
                 log_pipe_location_tag(&self->super.super.super.super));
       return FALSE;
     }
-
```

Comment:
I think a couple of empty lines help with readability, especially in such a long function.